src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/ExternalMovement.java

```diff
@@ -130,6 +130,11 @@ public static class PK implements Serializable {
     })
     private MovementType movementType;
 
+    @Column(name = "TO_ADDRESS_ID")
+    private Long toAddressId;
+
+    @Column(name = "FROM_ADDRESS_ID")
+    private Long fromAddressId;
 
```

Comment:
Leave these as just ids since they are not read in DPS. Also there is no way of knowing what type of address since this table has no owner class so there would be no way of mapping to concrete address sub class